src/components/TodosList.js

```diff
@@ -1,21 +1,28 @@
+/* eslint-disable import/no-named-as-default */
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/prop-types */
```

Comment:
Linter message is a way to ensure that we follow best practices. Instead of disabling linters, please try to fix the issues as that will make sure you have good readable code. For example; you can convert the class TodosList  to a function like below:


const TodosList = () => {}

instead of 

class TodosList extends Component { 
}